snare.py

```diff
@@ -79,6 +79,28 @@ def submit_slurp(self, data):
         except Exception as e:
             print(e)
 
+    def create_data(self, request):
+        data = dict(
```

Comment:
We can do it when sending errors, when we send normal requests, SNARE response depends on TANNER response., I can add response status field and make it None when sending normal request data